build/RedHat.dockerfile

```diff
@@ -1,11 +1,13 @@
-FROM registry.access.redhat.com/ubi8/go-toolset:latest as build
+FROM registry.access.redhat.com/ubi8/go-toolset:1.18 as build
 USER 0
 RUN mkdir /build
 WORKDIR /build
 COPY . .
 RUN make prep build strip
 
-FROM registry.access.redhat.com/ubi8/ubi-minimal
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+RUN mkdir /docs
+COPY docs/lighthouse_template.json /docs
```

Comment:
Please use Go file [embedding feature](https://pkg.go.dev/embed). We do this for everything - migrations and preload data. Then there is no reason to copy any files.